cogs/commands/pufferpanel.py

```diff
@@ -0,0 +1,62 @@
+import aiohttp
+import discord
+from discord import app_commands
+from discord.ext import commands
+from discord.ext.commands import Bot, Context, clean_content
+
+from config import CONFIG
+from utils import get_json_from_url as json_url
+
+LONG_HELP_TEXT = "Lists all the servers running on PufferPanel"
+
+SHORT_HELP_TEXT = "Server info from PufferPanel"
+
+
+class PufferPanel(commands.Cog):
+    def __init__(self, bot: Bot):
+        self.bot = bot
+
+    @commands.hybrid_command(help=LONG_HELP_TEXT, brief=SHORT_HELP_TEXT)
+    async def pufferpanel(self, ctx: Context):
```

Comment:
maybe change this to servers? or gaming_servers? or smthn (idk ask joey)

Reply:
Need to remove old server code also